WebDriverAgentLib/Categories/XCElementSnapshot+FBHelpers.m

```diff
@@ -157,6 +105,15 @@ - (XCElementSnapshot *)fb_parentCellSnapshot
     return targetCellSnapshot;
 }
 
++ (NSSet<NSNumber *> *)fb_getUniqueTypes:(NSArray<XCElementSnapshot *> *)matchingSnapshots
```

Comment:
We could make this method even more general by using `NSArray<id  <FBElement>> *` and move it to helper class for `FBElements`